SETUP_ARCH.md

```diff
@@ -0,0 +1,23 @@
+# Snowdrift setup instructions for Arch Linux
+
+Arch has the benefit of being very well-documented, so this document is
+very short!
+
+1.  [Install Haskell][1]. While you are at it, make sure you have the
+    `base-devel` group installed.
+2.  [Install PostgreSQL][2]. You don't need to go through setting up a
+    user or anything like that. Snowdrift will do that for you.
```

Comment:
Hmm. Let me think about it.

Maybe "you only need to go through the \"installing\" section,
everything thereafter is handled by our script". There's stuff after the
"setting up a user" section as well.

Peter Harpending

On Fri, May 22, 2015 at 05:49:54PM -0700, Aaron Wolf wrote:
> > @@ -0,0 +1,23 @@
> > +# Snowdrift setup instructions for Arch Linux
> > +
> > +Arch has the benefit of being very well-documented, so this document is
> > +very short!
> > +
> > +1.  [Install Haskell][1]. While you are at it, make sure you have the
> > +    `base-devel` group installed.
> > +2.  [Install PostgreSQL][2]. You don't need to go through setting up a
> > +    user or anything like that. Snowdrift will do that for you.
> 
> Ah, maybe reference it with wording like "you can skip all the stuff
> about setting up a user, our script handles that later".
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/snowdriftcoop/snowdrift/pull/310/files#r30940044